xcodeproj/internal/installer.template.sh

```diff
@@ -2,8 +2,51 @@
 
 set -euo pipefail
 
-readonly src="$PWD/%source_path%"
-readonly dest="$BUILD_WORKSPACE_DIRECTORY/%output_path%"
+# Functions
+
+# Echos the provided message to stderr and exits with an error (1).
+fail() {
+  local msg="${1:-}"
+  shift 1
+  while (("$#")); do
+    msg="${msg:-}"$'\n'"${1}"
+    shift 1
+  done
+  echo >&2 "${msg}"
+  exit 1
+}
+
+# Process Args
+
+while (("$#")); do
+  case "${1}" in
+    "--source")
```

Comment:
Is source needed as well? Shouldn't it always be from the exec root?

Reply:
My bad. You're correct. I was looking at the runfiles directory when I put this together. The test's initial PWD is under the `com_github_buildbuddy_io_rules_xcodeproj` directory. I will remove it.